airbyte-ci/connectors/connectors_insights/README.md

```diff
@@ -0,0 +1,57 @@
+# Connectors Insights
+
+Connectors Insights is a Python project designed to generate various insights from analysis of our connectors code. This project utilizes Poetry for dependency management and packaging.
```

Comment:
Funny enough, I think it's even okay to make those public at some point.

Reply:
They are somewhat already public as the bucket to which they are uploaded is the connector metadata service one which is open to the internet.